test/conformance/CMakeLists.txt

```diff
@@ -8,9 +8,18 @@ set(UR_CONFORMANCE_TEST_DIR ${CMAKE_CURRENT_SOURCE_DIR})
 function(add_test_adapter name adapter)
     set(TEST_TARGET_NAME test-${name})
     set(TEST_NAME ${name}-${adapter})
+
     add_test(NAME ${TEST_NAME}
-        COMMAND ${TEST_TARGET_NAME}
-        WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR})
+        COMMAND ${CMAKE_COMMAND}
+        -D TEST_FILE=${Python3_EXECUTABLE}
```

Comment:
hmm.. can we say it's a temporary solution, until all CTS tests will be fixed.

I believe we should make sure all contributors know that we should fix these tests (and update match files).. perhaps some note in contrib guide? or readme?